### DISCUSSION TITLE
The generation does not start

### DISCUSSION THREAD

**Discussion by Vigle-N0ir** [nftchef/art-engine#272] [Help + Q&A] [answered]
Hello first of all, thank you for your work.

Here is the error that happens at the beginning

PS C:\Users\Administrator\Desktop\Generate V3> node index.js
C:\Users\Administrator\Desktop\Generate V3\src\config.js
genator started undefined
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Back/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Body/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Face/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Earring/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Hair/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Mask/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Hat/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Shoes/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Top/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Bottom/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Shield/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Weapons/Thumbs.db Error: Unsupported image type
failed to load C:\Users\Administrator\Desktop\Generate V3\layers/Pets/Thumbs.db Error: Unsupported image type
C:\Users\Administrator\Desktop\Generate V3\src\main.js:345
  layerctx.drawImage(
           ^

TypeError: The first argument must be an object
    at drawElement (C:\Users\Administrator\Desktop\Generate V3\src\main.js:345:12)
    at C:\Users\Administrator\Desktop\Generate V3\src\main.js:735:7
    at Array.forEach (<anonymous>)
    at paintLayers (C:\Users\Administrator\Desktop\Generate V3\src\main.js:728:21)
    at C:\Users\Administrator\Desktop\Generate V3\src\main.js:851:11
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async startCreating (C:\Users\Administrator\Desktop\Generate V3\src\main.js:844:9)
PS C:\Users\Administrator\Desktop\Generate V3>

what should I do ?
Thanks in advance

**Comment by nftchef** [comment, marked as answer]
Issues like this are caused by non-image files being in the layer directories supplied to the generator config. 
in this case, the `Thumbs.db Error: Unsupported image type` `.db` file
